### PR TITLE
Implement event subscription

### PR DESCRIPTION
**Pull request by firefinchdev**

### CODE REVIEW

**Comment by firefinchdev** [comment]
@championswimmer Pls merge #353 . This PR depends on it.

**Comment by firefinchdev** [comment]
@championswimmer 
I have completed this PR, but it requires rebase and merging, which can be done after #353 is merged.
Pls review #353 and this PR.

File: src/routers/pages/client.js [hunk 1]
```diff
@@ -63,7 +68,48 @@ router.get('/:id/edit',
...
+                cAddress: '',
+                uAddress: '',
+                dAddress: '',
+                cClient: '',
+                uClient: '',
+                dClient: ''
+            }
+
+            eventSubscription.forEach (event => {
+                if (event.model === 'user') {
```
Comment (championswimmer) [inline src/routers/pages/client.js]: why not switch cases ?

Comment (firefinchdev) [reply src/routers/pages/client.js]: Done

**Comment by firefinchdev** [comment]
@championswimmer 
Changes done.
Pls review.

File: src/events/eventHelper.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  webhookURLs.forEach((webhookURL => {
+    request({
+      method: 'POST',
+      uri: webhookURL,
+      body: options,
+      json: true
+    }).then(parsedBody => {
+      console.log(parsedBody)
+    }).catch(err => {
+      console.error('webhookPOST', err)
```
Comment (championswimmer) [inline src/events/eventHelper.js]: put this into Raven please

File: src/events/users.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    .then((webhookURLs) => {
+      webhookPOST(webhookURLs, {
+        type: "CREATED",
+        model: "user",
+        success: true,
+        id: userId,
+        userId: userId
+      })
+    })
+    .catch(e => console.error('eventUserCreate', e))
```
Comment (championswimmer) [inline src/events/users.js]: all the errors should be Raven.captureException()

**Comment by firefinchdev** [comment]
Done.

File: src/events/users.js (hunk 3 above)
Comment (firefinchdev) [reply src/events/users.js]: Done.

File: src/events/eventHelper.js (hunk 2 above)
Comment (firefinchdev) [reply src/events/eventHelper.js]: Done.

**Pull request closed without merging**